apps/emqx_dashboard/test/emqx_dashboard_monitor_SUITE.erl

```diff
@@ -198,8 +195,6 @@ t_inplace_downsample(_Config) ->
     ok = gen_server:call(emqx_dashboard_monitor, dummy, infinity),
     All1 = emqx_dashboard_monitor:all_data(),
     All = drop_dummy_data_points(All1),
-    AllSent = lists:map(fun({_, #{sent := S}}) -> S end, All),
-    ?assertEqual(Total, lists:sum(AllSent)),
```

Comment:
We shouldn't assert anything about the sum, because these are not individual data points, but "snapshots" of the counters at given points in time.